coil-base/src/main/java/coil/fetch/UriFetcher.kt

```diff
@@ -56,20 +56,26 @@ internal class UriFetcher(
         )
     }
 
-    /** Return the asset's filename if [uri] must be handled by [AssetManager]. Else, return null. */
+    /** Return the asset's path if [uri] must be handled by [AssetManager]. Else, return null. */
     @VisibleForTesting
-    internal fun extractAssetFileName(uri: Uri): String? {
+    internal fun extractAssetPath(uri: Uri): String? {
         if (uri.scheme != ContentResolver.SCHEME_FILE) {
             return null
         }
 
         val segments = uri.pathSegments
-        return if (segments.count() == 2 &&
-            segments[0] == ASSET_FILE_PATH_SEGMENT &&
-            segments[1].isNotBlank()) {
-            segments[1]
-        } else {
-            null
+        if (segments.count() < 2 || segments[0] != ASSET_FILE_PATH_SEGMENT) {
+            return null
         }
+
+        val path = segments
+                .drop(1)
```

Comment:
Nit: Coil uses 4 spaces for continuation indent.

Reply:
Oops, I used IDE defaults for reformatting. Fixed.